src/test/java/com/browserstack/examples/config/WebDriverFactory.java

```diff
@@ -48,7 +48,7 @@
     private static final String DEFAULT_BUILD_NAME = "browserstack-examples-junit4";
 
     private static WebDriverFactory instance;
-    private final WebDriverConfiguration webDriverConfiguration;
+    public final WebDriverConfiguration webDriverConfiguration;
```

Comment:
Why is this public?

Reply:
I was using one of the method of WebDriverConfig class and hence to initialise this object in other class , i made it public, but now i have made it private and added that method in webdriverfactory class